Description: Disable detection of cygwin under Debian installs
 as this is not compatible with the way the environment is 
 managed through alternatives.
 .
 TODO - review alongside use of alternatives for managing 
 configuration.
Author: Akira Kitada <akitada@gmail.com>
Forwarded: not-needed

--- a/bin/zkServer.sh
+++ b/bin/zkServer.sh
@@ -92,15 +92,6 @@
     ZOOCFG="$2"
 fi
 
-if $cygwin
-then
-    ZOOCFG=`cygpath -wp "$ZOOCFG"`
-    # cygwin has a "kill" in the shell itself, gets confused
-    KILL=/bin/kill
-else
-    KILL=kill
-fi
-
 echo "Using config: $ZOOCFG" >&2
 
 case "$OSTYPE" in
@@ -179,7 +170,7 @@
     then
       echo "no zookeeper to stop (could not find file $ZOOPIDFILE)"
     else
-      $KILL -9 $(cat "$ZOOPIDFILE")
+      kill -9 $(cat "$ZOOPIDFILE")
       rm "$ZOOPIDFILE"
       echo STOPPED
     fi
